crates/primitives/src/transaction/signature.rs

```diff
@@ -100,7 +100,26 @@ impl Signature {
         secp256k1::recover(&sig, hash.as_fixed_bytes()).ok()
     }
 }
-
+impl From<&Signature> for [u8; 65] {
```

Comment:
imo this kind of from impl should be a functions.

Reply:
done!